app/models/hyrax/collection_type.rb

```diff
@@ -30,5 +33,24 @@ def collections?
       # And return 'false' to display the delete confirmation modal.
       true
     end
+
+    def self.find_or_create_default_collection_type
+      unless exists?(machine_id: DEFAULT_ID)
+        create_default_collection_type(machine_id: DEFAULT_ID, title: DEFAULT_TITLE)
+      end
+      DEFAULT_ID
+    end
+
+    def self.create_default_collection_type(machine_id:, title:)
+      create(machine_id: machine_id, title: title) do |c|
+        c.nestable = false
+        c.discoverable = true
+        c.sharable = true
+        c.allow_multiple_membership = true
+        c.require_membership = false
+        c.assigns_workflow = false
+        c.assigns_visibility = false
+      end
+    end
```

Comment:
Add a basic description for the User Collection.

"A User Collection can be created by any user allowing them to organize their works."